backend/src/main/java/org/eclipse/tractusx/puris/backend/DataInjectionCommandLineRunner.java

```diff
@@ -121,6 +147,8 @@ private void setupCustomerRole() throws JsonProcessingException {
         log.info(String.format("UUID of supplier partner: %s", supplierPartner.getUuid()));
         supplierPartner = partnerService.findByUuid(supplierPartner.getUuid());
         log.info(String.format("Found supplier partner: %s", supplierPartner));
+        supplierPartner = partnerService.findByBpns(supplierPartner.getSites().stream().findAny().get().getBpns());
```

Comment:
Currently we only support one site, don't we? How about findFirst instead of findAny? Same would apply for all usages.

Reply:
This is really just a shortcut to somehow make this sample in the DataInjectionCommandLineRunner work. I don´t think it´s sensible to assume anything in general here.

Reply:
You use findFirst at another point, too. For this class in particular you're right, but please do write code that reacts reproducable and the same way at different positions. It makes effects of changes more predictable and follows the open-closed principle.